build/Dockerfile

```diff
@@ -1,26 +1,45 @@
-FROM centos:centos8
+FROM quay.io/centos/centos:stream8
 
 # Use this build arg to set any default test script arguments
-ARG RUN_SCRIPT_ARGS=
 ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
-
+ENV ROBOT_EXTRA_ARGS=${ROBOT_EXTRA_ARGS}
 ENV HOME /tmp
 
+## Install supporting libraries and driver
 RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
-    dnf install -y git python3 unzip chromium chromedriver &&\
+    dnf install -y python38 jq git unzip chromium chromedriver redhat-lsb-core &&\
```

Comment:
Can we use a place somewhere in our internal Red Hat environment for downloading and installing all these packages?
From a bad experience, sometimes these remote sites fail and it will cause issues. If we need the latest versions (which will happen once in a few months) we can update them in our environment from their original places.
WDYT?

Reply:
Sure @ikanias I will look for a yum repo from redhat containing these libraries

Reply:
OK thanks!